### DISCUSSION TITLE
Deep Sea Mines as a level hazard

### DISCUSSION THREAD

**Discussion by Solbusaur** [FakeFishGames/Barotrauma#5840] [Ideas and suggestions]
An idea for a level hazard that is only found in Jovian Separist waters. Functions a little similarly to the ice spires, a jutting structure that wreaks havoc on the submarine if not avoided or taken care of.
The actual payload does not remain still by itself, it is secured to a nearby rock by a long chain. If there is a current, the payload is pushed with it a little. When the chain is destroyed, the payload will float upwards until it hits something.
The payload is activated if a character or a hull comes into contact with the "kill zone", a small radius sitting just above the surface of the payload. I can't suggest an explosion strength, I'm not too familiar with the explosion values. The chain does not collide with characters or the hull, but it can be cut or shot like the ice spires. The mine can also be avoided by detonating it with some gunfire at a safe distance.
The mine itself is hard to detect on sonar given its small size, the easiest way to see it is to have a periscope operator keep an eye out for them, they should have red lights on their surface. If a bot is controlling the periscope, they should not fire at the payload to disarm it until the submarine is far away enough.
I would also suggest a type that inflicts an EMP effect around it.
![image](https://user-images.githubusercontent.com/63509011/118840652-8126cc80-b8bf-11eb-966e-9f586d7429ae.png)

**Comment by ubertpendragon** [comment]
I made a (mostly) functional naval mine spawned as a wreck-shuttle combo, tried and tested

As soon as the mine deviates from a vertical wifi line, it explodes, which means any ship that bumps into it will detonate it. Unforunately it can also bump on ice spires, and sometimes enemies 

It's a lot of fun ramming into those, and I wish we had them in-game
[Naval_Mine_Dropper.zip](https://github.com/Regalis11/Barotrauma/files/6511938/Naval_Mine_Dropper.zip)
